ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -239,6 +239,45 @@ function cco_annotation(){
     fi
 }
 
+function disable_boot_image_update() {
+    # Get current machineManagers value
+    local current_value
+    current_value=$(oc get MachineConfiguration cluster -n openshift-machine-config-operator -o jsonpath='{.spec.managedBootImages.machineManagers}' 2>/dev/null)
+    local get_status=$?
+
+    if [ $get_status -ne 0 ]; then
+        echo "Error: Failed to get current MachineConfiguration. Check cluster access."
+        return 1
```

Comment:
It's better to set UPGRADE_FAILURE_TYPE before each error exit to help identifying upgrade failure.

Reply:
so the most challenging thing is which team owns this failure :)

Reply:
anyway I guess it should be MCO, fair enough.